core/modules/installer/installer.pages.inc

```diff
@@ -846,7 +846,7 @@ function installer_browser_get_install_list() {
  */
 function installer_browser_install_button_form($form, &$form_state) {
   $project_type = installer_browser_type_from_path();
-  $form['#attributes']['class'] = 'installer-browser-install-button-form';
+  $form['#attributes']['class'] = array('installer-browser-install-button-form');
```

Comment:
Funny, I just realized, that the line isn't necessary at all.

That class gets set anyway in function backdrop_retrieve_form, via `$form['#attributes']['class'][] = backdrop_html_class($form_id);`

Reply:
Interesting! Let's remove that, then!

Reply:
I can confirm that removing this line works too 😄